rclpy/rclpy/node.py

```diff
@@ -244,44 +239,25 @@ def destroy_node(self):
         if self.handle is None:
             return ret
 
-        # ensure that the passed node contains a valid capsule
-        class_ = self.handle.__class__
-        if not class_ or class_.__name__ != 'PyCapsule':
-            raise ValueError('The node handle must be a PyCapsule')
+        for sub in self.subscriptions:
+            _rclpy.rclpy_destroy_node_entity(sub.subscription_handle, self.handle)
```

Comment:
The return value is not being considered anymore.

Reply:
This is intentional. Currently `rclpy_destroy_node_entity` returns True on success or raises a RuntimeError on failure, so I got rid of the return value altogether.

Reply:
I see. If the return value of the function doesn't have any purpose anymore I would suggest to return `None` instead (like any Python function without a return statement).

Reply:
I believe that is what this PR is doing, raising and returning NULL on failure otherwise returning None:
https://github.com/ros2/rclpy/pull/129/files#diff-80dda03110b5be606b823d6b0558b5c2R1567

Reply:
Sorry for the noise. I don't know what I was looking at.